.github/scripts/upload.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Upload a binary to a bucket, supports dry-run mode
```

Comment:
Should we rename this file to ``upload_aws_ossci.sh`` ?

Reply:
Done